.npmignore

```diff
@@ -0,0 +1 @@
+test
```

Comment:
👀

Reply:
We shouldn't bundle the tests in the NPM package